src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,77 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_session")
```

Comment:
```suggestion
@Table(name = "movie_sessions")
```
you can also change `movie` to `movies` - we will fix this

Reply:
edited ✔️